.github/copilot-instructions.md

```diff
@@ -2,6 +2,42 @@
 
 > Instructions for GitHub Copilot on following JSM frontend coding standards across all repositories.
 
+## Role Switching Convention
+
+- always write in the chat which role you are using, considering the rule as follows:
+- [C-FE] = Frontend Specialist
+- [C-akamai] = Cache and Akamai Specialist
```

Comment:
It might be useful considering the project context. Ex:

- using a chat on web-customer and its context
- question/issues regarding CDN/Akamai